gym_roboy/envs/tests/test_msj_env.py

```diff
@@ -99,5 +102,24 @@ def test_msj_env_reward_is_lower_with_joint_vel_penalty():
     assert reward_with_no_joint_vel_penalty > reward_with_joint_vel_penalty
 
 
+def test_msj_env_agent_gets_bonus_when_reaching_the_goal():
+
+    msj_env = MsjEnv(is_agent_getting_bonus_for_done=False)
+    obs = msj_env.reset()
+    current_joint_angle = obs[0:3]
+    msj_env._set_new_goal(goal_joint_angle=current_joint_angle)
+    zero_action = np.array([0]*8)
```

Comment:
8 is here a magic number. Please replace with `msj_env.ACTION_DIM` or similar.

Reply:
done